### PR TITLE
I3183 add taiko klaytn fantom

### PR DESCRIPTION
**Pull request by marclllaks**
Adds:
- Taiko with wallet activity
- Klaytn
- Fantom wallet activity

### CODE REVIEW

File: public/networkLogos/taiko.svg
Comment (BrickheadJohnny) [inline public/networkLogos/taiko.svg:1]: We already have a `taiko-katla.svg`, it's the same image, just with white background, could we use that instead?

File: src/hooks/useTokens.ts [hunk 1]
```diff
@@ -74,6 +74,8 @@ export const TokenApiURLs: Record<Chain, string[]> = {
   OP_BNB: [],
   FORM_TESTNET: [],
   CYBER: [],
+  TAIKO: [],
+  KLAYTN: ["https://tokens.coingecko.com/klaytn/all.json"],
```
Comment (BrickheadJohnny) [inline src/hooks/useTokens.ts:78]: I get an `AccessDenied` error for this 👀

File: public/networkLogos/taiko.svg
Comment (marclllaks) [reply public/networkLogos/taiko.svg:1]: Sure thing, nice catch

File: src/hooks/useTokens.ts (hunk 1 above)
Comment (marclllaks) [reply src/hooks/useTokens.ts:78]: Yep, this is wrong indeed. We don't have a token list then

**Review by BrickheadJohnny — APPROVED** [review]
lfg

**Pull request merged**